admin/provisioner/clickhousestatic/provisioner.go

```diff
@@ -31,6 +31,21 @@ type Spec struct {
 	// ENV variable that contains the clickhouse DSN.
 	// This is an alternative to specifying the DSN directly, which can be useful for injecting secrets
 	DSNEnv string `json:"dsn_env"`
+
+	// ReadDSN with admin permissions for read operations (optional separate read replica)
+	ReadDSN string `json:"read_dsn,omitempty"`
+	// ReadDSNPath with admin permissions for read operations (optional separate read replica)
+	ReadDSNPath string `json:"read_dsn_path,omitempty"`
+	// ReadDSNEnv with admin permissions for read operations (optional separate read replica)
+	ReadDSNEnv string `json:"read_dsn_env,omitempty"`
```

Comment:
Do we use both `dsn_path` and `dsn_env`? I think it's just `dsn_env` now, right? I think we should just remove the one we don't use to keep things simple.